### ISSUE TITLE
Undefined variable

### ISSUE THREAD

**Issue by scisci** [tools/godep#230]
I am trying to compile a project that has github.com/btcsuite/btcd as a dependency. Everything builds fine with regular go commands but when I try godep go install, one of the global variables goes missing and I get:

Godeps/_workspace/src/github.com/btcsuite/btcd/btcec/precompute.go:25: undefined: secp256k1BytePoints

I can see that that variable is defined in another file in the GoDeps path:
(from gensecp256k1.go)
var secp256k1BytePoints = ""

But for some reason godep can't find that declaration or is dropping it.

**Comment by freeformz** [comment]
You need to run go generate inside the workspace. 

Try this...

```bash
cd <your repo>
OGOPATH=$GOPATH
GOPATH=$(godep path)
cd Godeps/_workspace/src/github.com/btcsuite/btcd/btcec
go generate 
GOPATH=$OGOPATH
cd <root of your repo>
godep go install ./...
```

You will have to check the generate code in though, which you may not want to do.

Your other option would be to wrap the build in a Makefile. This is because `go generate` is meant to be run before building **by design**.

**Comment by kr** [comment]
I think this is actually #160.

**Comment by freeformz** [comment]
@kr Gah, you are correct. I fixed it in #231. btcd already ran and saved the go generated bits, which were truncated silently by that bug, but would have been fixed by my example above (AFAICT).
